app/src/main/kotlin/org/cru/godtools/ui/tools/ToolsListFragment.kt

```diff
@@ -116,12 +115,14 @@ class ToolsListFragment() : BasePlatformFragment<ToolsFragmentBinding>(R.layout.
 
     private fun trackInAnalytics() {
         when (mode) {
-            MODE_ALL -> eventBus.post(AnalyticsScreenEvent(SCREEN_ALL_TOOLS))
```

Comment:
@frett is MODE_ALL no longer being used.

Reply:
https://github.com/CruGlobal/godtools-android/pull/2246/commits/00a6048cd71a3a744bb1c213a47cd73f7af32a11

yes, it was no longer needed after the all tools UI was switched to the new ToolsFragment.